### DISCUSSION TITLE
Clean HDMI after Restart via QR code

### DISCUSSION THREAD

**Discussion by VulcanCCIT** [gopro/labs#131] [General]
So I changed the firmware to the GoPro Labs firmware for the GoPro 11, I found online (as well as created my own QR code) for the !MHDMI=1  - clean HDMI QR code
 
I have put this QR Code in…..it gave me the red check mark but I do not think it accepted it. I still had to manually use the mode/select method to clear the icons on the screen. 
 
NOTE, when I did use it, the Gpro 11 Black was not hooked to HDMI at the time… do I need to have on HDMI for the QR code to work? Also, will it save this clean HDMI setting even after power down and power up? This is for a radio station to stream video of our DJs 24/7 and we just want the Clean HDMI to be there even after power up.  Basically trying to create an idiot/no human around to fix proof this GoPro.

Thank you all!

**Comment by dnewman-gpsw** [comment]
All !Mxxxx actions are saved.  Capture any clip, and load into https://gopro.github.io/labs/control/metadata/  which will display all the metadata stored.  Labs metadata is in the "User Metadata" 
![image](https://user-images.githubusercontent.com/10856084/198897337-a48386ff-bb9c-4e12-a80e-d430a6adca47.png)

This action is saved whether on not the HDMI or even MediaMod is connected.  

Look for a typo, as it worked in my HERO11 HDMI testing.

**Reply by VulcanCCIT** [reply]
ok, it is clear to me I am missing a step... when you say capture a clip then load to that URL...   all I did is turn on the GoPro, point it to the QR code, and that was it.  I am assuming, (which I could be totally wrong), that when I do that, and get the red check mark, that the GoPro captured the clip... how then do you upload it to that site?  is this all via the GoPro Quik app? or via the camera itself?  Again assuming, the camera is perhaps always looking at the URL and hits the URL upon boot up?    Thank you again for your time sir.

**Comment by dnewman-gpsw** [comment]
You are way over thinking this, just press record, it creates a clip, correct? Use computer, load the clip from the sd card into the web site I listed and press this button 
![image](https://user-images.githubusercontent.com/10856084/198898892-b8646300-1727-4383-9ace-d3abad2e1196.png)

**Comment by VulcanCCIT** [comment]
I redid the code via the cell phone app and it works great after power up.  I also told it to default to video.  Thank you for all of your help!
